ElasticsearchReportHelper.php

```diff
@@ -380,9 +380,8 @@ public static function download(array $options) {
     // a select control that will be used to indicate the selected columns template.
     if (!empty($options['columnsTemplate']) && is_array($options['columnsTemplate'])) {
       $availableColTypes = array(
-        "default" => lang::get("Standard download format"),
-        "easy-download" => lang::get("Backward-compatible format"),
-        "mapmate" => lang::get("Mapmate-compatible format"),
+        "easy-download" => lang::get("Standard download format"),
```

Comment:
Am I correct in that removing default won't affect existing sites here, because the columns template chooser isn't live yet anywere?

Reply:
That's my thinking John.